indico/migrations/versions/20230928_1703_d8562ad31e90_add_created_dt_on_user_model.py

```diff
@@ -0,0 +1,26 @@
+"""Add created_dt on User model
+
+Revision ID: d8562ad31e90
+Revises: b45847c0e62f
+Create Date: 2023-09-28 17:03:58.549354
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+from indico.core.db.sqlalchemy import UTCDateTime
+
+
+# revision identifiers, used by Alembic.
+revision = 'd8562ad31e90'
+down_revision = 'b45847c0e62f'
```

Comment:
this is wrong. probably created while on 3.2? ;)
on master it should be `0c44046dc1be`